src/components/DropdownList.tsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
bad idea, better to leave comments right before you gonna use them. But what's even better - not to disable linting rules, just fix them instead